include/mercury/kes_dashboard_live_tiles_cli/OnlineLoader.h

```diff
@@ -52,13 +52,13 @@ class OnlineLoader : public IOnlineLoader {
     double getQueryCooldown() const override;
 
  public:  // Constants.
-    static constexpr const char* KES_DLT_URL =
-        "https://ws.os.kes.kessandbox.co.uk/";
+    static constexpr const char* KES_DLT_URL = "https://dlt.os.kes.kano.me/";
+    // static constexpr const char* KES_DLT_URL = "https://dlt.os.kes.kessandbox.co.uk/";
```

Comment:
Do we need this at all?

Reply:
It was useful when making the CNAMEs and switching between. Will have to move those out into a config file anyway at some point.